### DISCUSSION TITLE
可否在“播放器触摸支持”下添加每项功能的开关（亮度，音量，三档播放速度）

### DISCUSSION THREAD

**Discussion by chester0326** [the1812/Bilibili-Evolved#1964] [功能建议 / Ideas]
提出这一点，主要是因为我设置了播放器置顶和默认播放器模式为宽屏，而这样的话网页加载好之后总是会自动往下滚一点，然后有一部分播放器不在窗口内：
![image](https://user-images.githubusercontent.com/52564088/122631190-e412ab80-d097-11eb-9d38-0ed52e9f429a.png)
而我在往上划回去的时候经常会滑到亮度。同样，往下划看评论的时候也经常划错。然后因为手动调整还很难回到100亮度（强迫症很难受），不调到100还有点影响画面观感，所以希望能自己禁用亮度调整。而既然都提出这个了，不如把所有功能都设置成可以自己手动调吧，所以起了这个标题。

**Comment by dvd6** [comment]
同意呀，这个亮度是调播放器亮度，不是屏幕背光就比较鸡肋，也容易误触，超过100有副作用，希望可以禁用掉
